OutSystems/outsystems-datagrid-reactive
# Add configs files to Visual Code

This PR is for improving the simplicity of setting up the development environment.

### What was done
* Added file with the recommended extensions for vscode
* Added file with the default configurations to vscode
* Added a setup command in the package.json - runs `npm i && tsc`

### Test Steps
1. Delete the repo from your machine & uninstall all extensions
2. Download the repo, run `npm run setup`
3. What should happen?
    a. Vscode should recommend to install the extensions.
    b. Everything that is required to compile the code should be installed.

### Checklist
